controls/entry/text-input.md

```diff
@@ -0,0 +1,34 @@
+---
+title: TextInput
+page_title: .NET MAUI TextInput Documentation - TextInput
+description: Learn more about the TextInput control used inside the Telerik UI for .NET MAUI Entry controls.
+position: 25
+slug: entry-textinput
+---
+
+# TextInput
+
+The TextInput for .NET MAUI is a control which inherits from the Microsoft .NET MAUI `Entry` control. The Telerik UI for .NET MAUI `RadEntry` control uses the TextInput control in its control template for the text input. The `RadEntry` is a templated control, with a `ControlTemplate`, and in that `ControlTemplate` there are borders, close button, and a TextInput.
```

Comment:
It gets confusing when the text switches between class names and component names for the same thing. Stick to one of them.